launch/docker.go

```diff
@@ -282,7 +334,8 @@ func (d *docker) runBuild(buildEntry buildEntry) error {
 			{".", "/tmp/sd-local.env"},
 			{"set", "+a"},
 			{"export", "PS1='sd-local# '"},
-			{"sdrun() { . /$SD_UTILS_DIR/bin/sd-run $@; }"},
+			{"sdrun() { . /$SD_UTILS_DIR/bin/sdrun $@; }"},
+			{"source /$SD_UTILS_DIR/bin/sdrun-bash-completion"},
```

Comment:
Probably, `source`  command is not available in some docker image with using `/bin/sh`.
You need to use `.` (dot) command.

Reply:
fixed